vehicle/tests/golden/error/tensorLike/invalidTensorLikeType/TypeCheck.err.golden

```diff
@@ -0,0 +1,6 @@
+Error in file 'spec.vcl' at Line 10, Columns 5-7: The type of @tensor 'f3':
+  R1
+is not supported. All fields of a record annotated with '@tensor' must have the same type and must be either:
```

Comment:
Hmm this error message is actually actively wrong. See comment in error generation above.